### PR TITLE
Add sensitive data collected

### PR DESCRIPTION
**Pull request by antonpirker**
Added tables documenting what data is collected by Django and FastAPI integrations. Internally used integrations (FastAPI uses Starlette and ASGI, Django uses WSGI and ASGI) have also been documented:

Django:
https://sentry-docs-git-antonpirker-python-sensitive-data-collected.sentry.dev/platforms/python/guides/django/

FastAPI:
https://sentry-docs-git-antonpirker-python-sensitive-data-collected.sentry.dev/platforms/python/guides/fastapi/

Starlette:
https://sentry-docs-git-antonpirker-python-sensitive-data-collected.sentry.dev/platforms/python/guides/starlette/

WSGI:
https://sentry-docs-git-antonpirker-python-sensitive-data-collected.sentry.dev/platforms/python/configuration/integrations/wsgi/

ASGI:
https://sentry-docs-git-antonpirker-python-sensitive-data-collected.sentry.dev/platforms/python/configuration/integrations/asgi/

### CODE REVIEW

File: src/platforms/python/common/configuration/integrations/wsgi.mdx [hunk 1]
```diff
@@ -48,6 +48,48 @@ Each request has a separate scope. Changes to the scope within a view, for examp
...
+
+### Data Collected by Performance Monitoring
+
+<Note>
+
+The parameter `traces_sample_rate` needs to be set when initializing the Sentry SDK to monitor performance.
+
+</Note>
+
+|                                                                                                                                                   | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT) | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                     |
```
Comment (lizokm) [inline src/platforms/python/common/configuration/integrations/wsgi.mdx:78]:
```suggestion
| Type of data collected                                                                                                                                            | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT) | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                     |
```

File: src/platforms/python/guides/django/index.mdx [hunk 2]
```diff
@@ -77,6 +78,54 @@ The parameter `traces_sample_rate` needs to be set when initializing the Sentry
 
 </Note>
 
+## Data Collected
+
+The Django integration uses the [WSGI Integration](/platforms/python/configuration/integrations/wsgi/) and for projects with Django Channels additionally the [ASGI Integration](/platforms/python/configuration/integrations/asgi/) internally.
```
Comment (lizokm) [inline src/platforms/python/guides/django/index.mdx:83]:
```suggestion
The Django integration uses the [WSGI Integration](/platforms/python/configuration/integrations/wsgi/). In addition, if projects use Django Channels, the [ASGI Integration](/platforms/python/configuration/integrations/asgi/) is used internally.
```

File: src/platforms/python/guides/django/index.mdx [hunk 3]
```diff
@@ -77,6 +78,54 @@ The parameter `traces_sample_rate` needs to be set when initializing the Sentry
...
+
+### Data Collected by Issue Reporting
+
+|                                                                                                                                                   | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT)                                                    | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (lizokm) [inline src/platforms/python/guides/django/index.mdx:87]:
```suggestion
| Type of data collected                                                                                                                                             | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT)                                                    | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                                                                                                                                                                                                                                     |
```

File: src/platforms/python/guides/django/index.mdx [hunk 4]
```diff
@@ -77,6 +78,54 @@ The parameter `traces_sample_rate` needs to be set when initializing the Sentry
...
+
+### Data Collected by Performance Monitoring
+
+<Note>
+
+The parameter `traces_sample_rate` needs to be set when initializing the Sentry SDK to monitor performance.
+
+</Note>
+
+|                                                                                                                                                   | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT)                                                    | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (lizokm) [inline src/platforms/python/guides/django/index.mdx:112]:
```suggestion
| Type of data collected                                                                                                                                                  | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT)                                                    | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                                                                                                                                                                                                                                     |
```

File: src/platforms/python/guides/fastapi/index.mdx [hunk 5]
```diff
@@ -126,20 +126,24 @@ The default is `"url"`.
...
-| Performance spans                                                                                                                               | <span style="color: #69C289;">Not Sent</span>                | <span style="color: #69C289;">Not Sent</span>                                                                                                                                 |
+|                                                                                                                                                   | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT)                                             | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                       |
```
Comment (lizokm) [inline src/platforms/python/guides/fastapi/index.mdx:133]:
```suggestion
| Type of data collected                                                                                                                                                 | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT)                                             | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                       |
```

File: src/platforms/python/guides/fastapi/index.mdx [hunk 6]
```diff
@@ -149,18 +153,20 @@ The parameter `traces_sample_rate` needs to be set when initializing the Sentry
...
-| Performance spans                                                                                                                               | <span style="color: #C83852;">Sent</span>                    | <span style="color: #C83852;">Sent</span>                                                                                                                                    |
+|                                                                                                                                                   | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT)                                             | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                      |
```
Comment (lizokm) [inline src/platforms/python/guides/fastapi/index.mdx:156]:
```suggestion
| Type of data collected                                                                                                                                               | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT)                                             | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                      |
```

File: src/platforms/python/guides/starlette/index.mdx [hunk 7]
```diff
@@ -99,3 +99,52 @@ In the above code, the transaction name will be:
...
 
 The default is `"url"`.
+
+## Data Collected
+
+The Starlette integration uses the [ASGI Integration](/platforms/python/configuration/integrations/asgi/) internally.
+
+### Data Collected by Issue Reporting
+
+|                                                                                                                                                   | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT) | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                      |
```
Comment (lizokm) [inline src/platforms/python/guides/starlette/index.mdx:109]:
```suggestion
| Type of data collected                                                                                                                                                | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT) | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                      |
```

File: src/platforms/python/guides/starlette/index.mdx [hunk 8]
```diff
@@ -99,3 +99,52 @@ In the above code, the transaction name will be:
...
+
+### Data Collected by Performance Monitoring
+
+<Note>
+
+The parameter `traces_sample_rate` needs to be set when initializing the Sentry SDK to monitor performance.
+
+</Note>
+
+|                                                                                                                                                   | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT) | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                     |
```
Comment (lizokm) [inline src/platforms/python/guides/starlette/index.mdx:132]:
```suggestion
| Type of data collected                                                                                                                                                 | Send PII: Off <br/>`(send_default_pii=False)` <br/>(DEFAULT) | Send PII: On <br/>`(send_default_pii=True)` <br/>&nbsp;                                                                                                                                     |
```

**Review by lizokm — APPROVED** [review]
Made a few additions.

**Comment by antonpirker** [comment]
Hey @lizokm !

Thanks for the review, but we have decided to not merge this because we will make easier to maintain documentation for this. Sorry for the "all for nothing" work.

To prevent this from happening in the future, please do not review "draft PRs" (they have a this grey "Draft" label top left)

**Comment by antonpirker** [comment]
Closing in favor of: https://github.com/getsentry/sentry-docs/pull/6378

**Pull request closed without merging**